web/src/compose.js

```diff
@@ -131,7 +131,7 @@ export function create_message_object() {
         content: compose_state.message_content(),
```

Comment:
// Changes here must also be kept in sync with echo.try_deliver_locally

I don't see a similar change in `echo.js` -- does this comment not apply in this case?

Reply:
Yeah the relevant changes are on the message object passed into that function